README.md

```diff
@@ -2,13 +2,13 @@
 
 # Project Name
 
```

Comment:
> Please make sure to have a clean and professional readme. More specifically: 
- [ ] Please remove headings that are not used, It will help users to focus on necessary information, this is what you currently have:
<img width="277" alt="Screenshot 2022-03-18 at 10 56 56" src="https://user-images.githubusercontent.com/77038610/158952199-5580c7df-3985-4bb0-851a-944131adef2f.png">

- [ ] Please customize `Getting started` section, this is what you currently have:
<img width="865" alt="Screenshot 2022-03-18 at 10 58 43" src="https://user-images.githubusercontent.com/77038610/158952406-5f13321c-48d9-442e-94b1-b7a42089415d.png">

- [ ] In the `Project Name` section please write some sentences describing the project and change the title to your project name, eg: `Portfolio`. 
- [ ] In the `Authours` section please add more links, for example, your linked link. 
- [ ] MIT license link is broken, please fix the link by adding the MIT file

`After all the changes, make sure that  the readme has no irrelevant information and broken links`